circle/functions

```diff
@@ -427,3 +427,38 @@ chart_update_version() {
     git commit -m "$CHART_NAME: bump chart version to \`$CHART_VERSION_NEXT\`" >/dev/null
   fi
 }
+
+get_image_build_dir() {
+    local RELEASE_SERIE=${1}
+    local BASE_IMAGE=${2}
+
+    if [[ -f $RS/Dockerfile ]]; then
+        IMAGE_BUILD_DIR=$RELEASE_SERIE
+    elif [[ -f $RS/$BI/Dockerfile ]]; then
```

Comment:
Lets prioritize `if [[ -f $RS/$BI/Dockerfile ]]; then`  over `if [[ -f $RS/Dockerfile ]]; then` as the latter is a fallback/backward compatibility thing